packages/core/src/common/_customs.scss

```diff
@@ -338,4 +340,52 @@ $g-wizard-10-500: #517F58;
 $g-wizard-10-600: #2E6537;
 $g-wizard-10-700: #26542E;
 $g-wizard-10-800: #1F4325;
-$g-wizard-10-900: #17331C;
\ No newline at end of file
+$g-wizard-10-900: #17331C;
+
+$g-lavander-25: #FCFBFD;
+$g-lavander-50: #FAF8FC;
+$g-lavander-100: #F2EDF7;
+$g-lavander-200: #E8DFF1;
+$g-lavander-300: #D4C3E5;
+$g-lavander-400: #C3AADA;
+$g-lavander-500: #AA86CB;
+$g-lavander-600: #9467BD;
+$g-lavander-700: #60437A;
+$g-lavander-800: #463159;
+$g-lavander-900: #2C1F38;
```

Comment:
I think we have a typo here... it should be "lavEnder" 😢 

![image](https://user-images.githubusercontent.com/36133677/188087664-2e64b2a7-c8a7-48e3-8422-15c6c4738eb4.png)